chain/chain/src/chain.rs

```diff
@@ -905,6 +905,9 @@ impl Chain {
     {
         let block_hash = *block.hash();
         let timer = metrics::BLOCK_PROCESSING_TIME.start_timer();
+        let success_timer = metrics::BLOCK_SUCCESSFUL_PROCESSING_TIME.start_timer();
+        // We measure time once, because the histogram measures time of all block processing
+        // attempts, and filtering needs to be done manually.
```

Comment:
I feel like this comment is better be placed on `pub static BLOCK_SUCCESSFUL_PROCESSING_TIME:`